tests/integration-tests/configs/develop.yaml

```diff
@@ -744,7 +744,7 @@ test-suites:
   trainium:
     test_trainium.py::test_trainium:
       dimensions:
-        - regions: ["use2-az3"]  # do not move, unless instance type support is moved as well
+        - regions: [{{ trn1_32xlarge_CAPACITY_RESERVATION_2_INSTANCES_2_HOURS }}]
```

Comment:
Non-blocking: Maybe later we can remove `instances` paramter too as you mention the instance_type you want in the regions name itself!